tests/foreman/ui_airgun/test_contenthost.py

```diff
@@ -100,8 +134,13 @@ def set_ignore_facts_for_os(value=False):
     ignore_setting.update({'value'})
 
 
+def run_remote_command_on_content_host(command, vm_module_streams):
+    result = vm_module_streams.run(command)
+    assert result.return_code == 0
+
+
 @tier3
-def test_positive_end_to_end(session, repos_collection, vm):
+def test_positive_end_to_end(session, repos_collection_for_module_streams, vm_module_streams):
```

Comment:
You move this test to run on RHEL8 repos instead of RHEL7 repos. So we get new coverage, but at expense of coverage that we already had. I'm not sure if this is good idea.

Possibly you could use `pytest.mark.parametrize` to run this test twice, for both RHEL7 repos and RHEL8 repos?

Reply:
@mirzal Module Streams tests can only run on rhel_8, we won't be requiring the rhel_7 repos. I think it is not a valid case for adding coverage.  If you see repos_collection_for_module_streams function that only contains custom repos, not even tools repo.

Reply:
@omkarkhatavkar You don't make any changes to this test - you only change parameters that you pass in. Before this PR, it run on default VM (with katello-agent) with repos: FAKE_1_YUM_REPO, FAKE_6_YUM_REPO, Tools and AnsibleEngine.

After this PR, it will run on new VM (without katello-agent) with only one repo: CUSTOM_MODULE_STREAM_REPO_2.

With this PR, we gain coverage for RHEL8 module streams repo, but at expense of all the repos that we used in the past. I'm not saying you should improve test to run module stream on RHEL7 - I'm saying that with this change, we lose coverage for RHEL7 repos we had in the past. And I'm not sure if this is good idea.

Reply:
My Bad! I accidentally updated the existing test. I will revert this.